packages/core/content-type-builder/admin/src/components/AttributeOptions/tests/index.test.js

```diff
@@ -0,0 +1,109 @@
+import { render, screen, getByText, fireEvent } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { lightTheme, darkTheme } from '@strapi/design-system';
+import LanguageProvider from '../../../../../../admin/admin/src/components/LanguageProvider';
+import Theme from '../../../../../../admin/admin/src/components/Theme';
+import ThemeToggleProvider from '../../../../../../admin/admin/src/components/ThemeToggleProvider';
+import en from '../../../../../../admin/admin/src/translations/en.json';
+import FormModalNavigationProvider from '../../FormModalNavigationProvider';
+import pluginEn from '../../../translations/en.json';
+import getTrad from '../../../utils/getTrad';
+import AttributeOptions from '../index';
+
+const mockAttributes = [
+  [
+    'text',
+    'email',
+    'richtext',
+    'password',
+    'number',
+    'enumeration',
+    'date',
+    'media',
+    'boolean',
+    'json',
+    'relation',
+    'uid',
+  ],
+  ['component', 'dynamiczone'],
+];
+
+const makeApp = () => {
+  const history = createMemoryHistory();
+  const messages = {
+    en: Object.keys(pluginEn).reduce(
+      (acc, current) => {
+        acc[getTrad(current)] = pluginEn[current];
+
+        return acc;
+      },
+      { ...en }
+    ),
+  };
```

Comment:
I'm not sure you really need this. If you render `LanguageProvider` without messages, doesn't it just print the `defaultMessage`? And if not: do we not have a utility somewhere to abstract this? It feels a bit odd to do so much setup for a single test like this one ...

Reply:
Good point, thanks. I updated. I had copied this setup from `packages/core/content-type-builder/admin/src/pages/ListView/tests/index.test.js` maybe in another PR we should refactor that to use `defaultMessage` as well 🤔